collect_app/src/main/java/org/odk/collect/android/formlists/savedformlist/SavedFormListItemView.kt

```diff
@@ -0,0 +1,35 @@
+package org.odk.collect.android.formlists.savedformlist
+
+import android.content.Context
+import android.view.LayoutInflater
+import android.widget.FrameLayout
+import android.widget.ImageView
+import android.widget.TextView
+import org.odk.collect.android.R
+import org.odk.collect.android.databinding.FormChooserListItemMultipleChoiceBinding
+import org.odk.collect.android.instancemanagement.getStatusDescription
+import org.odk.collect.forms.instances.Instance
+import java.util.Date
+
+class SavedFormListItemView(context: Context) : FrameLayout(context) {
+
+    val binding =
+        FormChooserListItemMultipleChoiceBinding.inflate(LayoutInflater.from(context), this, true)
+
+    var instance: Instance? = null
```

Comment:
Similar to `BlankFormListItemView`. Do we need to store the instance here? I don't think so and a simple setup method would suffice.